lib/spark.js

```diff
@@ -146,6 +145,14 @@ export default class Spark {
         }
 
         if (!stats.carTooLarge) {
+          try {
+            await verifyContent(cid, carBytes)
```

Comment:
This verification step is redundant when retrieving via Lassie because Lassie verifies the content for us.

I can skip the check when `protocol !== 'http'`, but I feel that's not worth the extra complexity.

WDYT?